python/cuml/neighbors/nearest_neighbors.pyx

```diff
@@ -853,7 +864,7 @@ class NearestNeighbors(Base,
             numpy's CSR sparse graph (host)
 
         """
-        if not self.X_m:
+        if not self._fit_X:
```

Comment:
This whole method (kneighbors_graph) is not being dispatched to the CPU. I may end up tackling this in 4908, but if not, that will need to be fixed before a merge.

Reply:
I took care of it in 4908

Reply:
True! Thanks for taking care of this one